cmd/btfs/functest.go

```diff
@@ -39,6 +40,17 @@ func get_functest(btfsBinaryPath string) error {
 	}
 
 	cmd := exec.Command(btfsBinaryPath, "get", "-o", dir, testfile)
+
+	go func() {
+		time.Sleep(timeoutSeconds * time.Second)
+		err := cmd.Process.Kill()
+		if err != nil {
+			if !strings.Contains(err.Error(), "process already finished") {
+				fmt.Printf("cannot kill process: [%v] \n", err)
```

Comment:
This message "cannot kill process" will show on terminal, we may need rephrase.
what't the consequences if the process still running? Like will it block the daemon start again? Do we need user to manually kill this process?